js/store.js

```diff
@@ -7,6 +7,7 @@ import arvic from './modules/arvic.js';
 const store = Vuex.createStore({
```

Comment:
Import your module first.

```suggestion
import jake from './modules/jake.js';

const store = Vuex.createStore({
```